barretenberg/cpp/src/barretenberg/dsl/acir_format/acir_to_constraint_buf.cpp

```diff
@@ -174,15 +177,22 @@ void handle_arithmetic(Program::Opcode::AssertZero const& arg, AcirFormat& af)
         // gate instead. We could probably always use a width-4 gate in fact.
         if (pt == poly_triple{ 0, 0, 0, 0, 0, 0, 0, 0 }) {
             af.quad_constraints.push_back(serialize_mul_quad_gate(arg.value));
+            af.original_opcode_indices.quad_constraints.push_back((opcode_index));
```

Comment:
Why do you need the double `(( ))` ?

Reply:
Remnant of old code that used tuples, removing!